### ISSUE TITLE
Fun with MSYS2 CLANG64

### ISSUE THREAD

**Issue by iahung** [objeck/objeck-lang#199]
I tested what happen if I use `deploy_msys2-ucrt.sh` instead of `deploy_msys2-clang.sh` on MSYS2 CLANG64.

Without `export OBJECK_STDIO=binary`:

```
$ ../bin/obc -src hello_0.obs
潃灭汩摥ㄠ挠慬獳ਮ

$ ../bin/obr hello_0
效汬⁯潗汲੤
```

With `export OBJECK_STDIO=binary`:

```
$ ../bin/obc -src hello_0.obs
潃灭汩摥ㄠ挠慬獳ਮ

$ ../bin/obr hello_0
Hello World

```

Who on earth could know what these texts mean? I fed them to Google Translate and it failed to give me answers!

**Comment by iahung** [comment]
I also found when using `deploy_msys2-ucrt.sh` instead of `deploy_msys2-clang.sh` on MSYS2 CLANG64, it failed to build the native dlls.

**Comment by objeck** [comment]
> I also found when using `deploy_msys2-ucrt.sh` instead of `deploy_msys2-clang.sh` on MSYS2 CLANG64, it failed to build the native dlls.

You'll need to use the ucrt64 shell

**Comment by iahung** [comment]
> > I also found when using `deploy_msys2-ucrt.sh` instead of `deploy_msys2-clang.sh` on MSYS2 CLANG64, it failed to build the native dlls.
> 
> You'll need to use the ucrt64 shell

Yeah I know. This test is only for fun :smile:

**Comment by objeck** [comment]
> I tested what happen if I use `deploy_msys2-ucrt.sh` instead of `deploy_msys2-clang.sh` on MSYS2 CLANG64.
> 
> 
> 
> Without `export OBJECK_STDIO=binary`:
> 
> 
> 
> ```
> 
> $ ../bin/obc -src hello_0.obs
> 
> 潃灭汩摥ㄠ挠慬獳ਮ
> 
> 
> 
> $ ../bin/obr hello_0
> 
> 效汬⁯潗汲੤
> 
> ```
> 
> 
> 
> With `export OBJECK_STDIO=binary`:
> 
> 
> 
> ```
> 
> $ ../bin/obc -src hello_0.obs
> 
> 潃灭汩摥ㄠ挠慬獳ਮ
> 
> 
> 
> $ ../bin/obr hello_0
> 
> Hello World
> 
> 
> 
> ```
> 
> 
> 
> Who on earth could know what these texts mean? I fed them to Google Translate and it failed to give me answers!

Try piping the output to a text file

**Comment by iahung** [comment]
> Try piping the output to a text file

The output is still the same as above :smile:

**Comment by objeck** [comment]
Fixed in af42131d0adcb1af35343c4b81ebc037a05aad34. Going to close as MSYS2 builds are released and the issue is only for MSYS2 Clang 64.
